geozero/src/csv/csv_writer.rs

```diff
@@ -24,12 +24,8 @@ impl<'w, W: Write> CsvWriter<'w, W> {
         }
     }
 
-    pub fn set_dims(&mut self, dims: CoordDimensions) {
```

Comment:
Hmm... I guess this still needs to be restored.

Reply:
Ok, I've restored the dimensions functionality with a new API. 

Note this will be a breaking change for CSVWriter, instead of `csv_writer.set_dims`, it's now part of the constructor, like with Wkt: `CSVWriter::with_dims`.